builtin/logical/transit/path_sign_verify.go

```diff
@@ -252,6 +264,7 @@ func (b *backend) pathSignWrite(ctx context.Context, req *logical.Request, d *fr
 
 	prehashed := d.Get("prehashed").(bool)
 	sigAlgorithm := d.Get("signature_algorithm").(string)
+	saltLength := d.Get("salt_length").(string)
```

Comment:
We should have some upfront validation in place for the new field and return a `logical.ErrorResponse` as we do a bit higher if the value is not acceptable.